corehq/apps/export/export.py

```diff
@@ -105,14 +105,14 @@ def open(self, export_instances):
             finally:
                 self.writer.close()
 
-    def write(self, table, row):
+    def write(self, table, row, hyperlink_column_indices):
```

Comment:
Adding this argument to every `write()` method bloats the function signature and requires modifications to a lot of code that simply passes it through and shouldn't need to know about it. It doesn't seem like the right level of abstraction—we would not want to add an argument like this for every different formatting option. Ideally this information be included in the `table` object. Would that be possible?

Reply:
Good point.  I updated the code to pass in the column indices with ```ExportRow```, which seems like the best place to do it.  I didn't use the table config in this spot because not all rows get the hyperlink formatting (ie header rows)